### ISSUE TITLE
Create a new section in `i_create_a_node.js` Cypress step definition file for the `campaign_landing_page` content type.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#12458]
See #11493.

## Acceptance Criteria
- [x] A section has been created in `i_create_a_node.js` corresponding to the specified content type.
- [x] A new test is added in `content_type__<content_type>.feature` utilizing the step definition to create and save a node of the specified content type.
- [x] Tests pass reliably (run the specific content type's test in a loop 20 times without failure).  This is probably most easily done by running a script like the following:
```bash
content_type=banner;
for i in $(seq 1 20); do
  echo "Running test # ${i}...";
  ./node_modules/.bin/cypress run --spec "tests/cypress/integration/behavioral/content_type__${content_type}.feature";
done;
```

**Comment by ndouglas** [comment]
This was actually closed by #12456, I just lost count.

**Comment by ndouglas** [comment]
This actually took more like a 5 😩
